server/src/services/ItemsService.js

```diff
@@ -0,0 +1,216 @@
+import { Items } from '../db/Models/item.model';
+import mongoose from 'mongoose';
+
+const langs = Object.freeze({
```

Comment:
Looks like code duplicate for constants, also, it worth to save list of languages in DB ( as a part of admin tool )